src/App.tsx

```diff
@@ -158,6 +158,7 @@ function App() {
   });
 
   const [campusCoordinate, setCampusCoordinate] = useState<Campus>(campusData[0]);
```

Comment:
Since this is of type "Campus" doesn't it already have the icon in the object? I would recommend changing this variable name to `campus, setCampus` and then in the code below, you can do something like:

```
<Marker longitude={campus.longitude} latitude={campus.latitude} anchor="bottom" >
  <img src={campus.img} />
</Marker>
```